MM-32943 add promote/demote user to local router (#4459)
mattermost/docs: source/administration/mmctl-cli-tool.rst

Child Commands
  -  `mmctl user activate`_ - Activate a user
  -  `mmctl user create`_ - Create a user
  -  `mmctl user deactivate`_ - Deactivate a user
  -  `mmctl user delete`_ - Delete a user
  -  `mmctl user deleteall`_ - Delete all users and all posts (local command only)
  -  `mmctl user demote`_ - Demote a user to a guest
  -  `mmctl user email`_ - Set user email
  -  `mmctl user invite`_ - Invite a user
  -  `mmctl user list`_ - List users

>   -  `mmctl user promote`_ - Promote a guest to a user

Reviewer: ```suggestion
  -  `mmctl user promote`_ - Promote a guest to user
```

>   -  `mmctl user reset_password`_ - Reset a user password

Reviewer: ```suggestion
  -  `mmctl user reset_password`_ - Reset a user's password
```

   --firstname string   Optional. The first name for the new user account
   -h, --help           help for create
   --lastname string    Optional. The last name for the new user account

>    --guest              Optional. If supplied, the new user will be a guest. Defaults to false.

Reviewer: ```suggestion
   --guest              Optional. If supplied, the new user will be a guest. (default "false")
```

   --local                       allows communicating with the server through a unix socket
   --strict                      will only run commands if the mmctl version matches the server one

mmctl user demote
^^^^^^^^^^^^^^^^^

**Description**

> Demote a user to a guest.

Reviewer: ```suggestion
Demote a user to guest.
```

   --local                       allows communicating with the server through a unix socket
   --strict                      will only run commands if the mmctl version matches the server one

mmctl user promote
^^^^^^^^^^^^^^^^^^

**Description**

> Promote a guest to a user.

Reviewer: ```suggestion
Promote a guest to user.
```

   --local                       allows communicating with the server through a unix socket
   --strict                      will only run commands if the mmctl version matches the server one

mmctl user verify
^^^^^^^^^^^^^^^^^

**Description**

    Verify the email address of a new user.

**Format**

.. code-block:: none

   mmctl user verify [users] [flags]

**Example**
    
.. code-block:: none

   mmctl user verify user1
     
**Options**

.. code-block:: sh

    -h, --help       help for email

**Options inherited from parent commands**

.. code-block:: sh

>       --config-path string           path to the configuration directory. If "$HOME/.mmctl" exists it will take precedence over the default value (default "$XDG_CONFIG_HOME")

Reviewer: ```suggestion
      --config-path string       path to the configuration directory. If "$HOME/.mmctl" exists it will take precedence over the default value (default "$XDG_CONFIG_HOME")
```

   --format string               the format of the command output [plain, json] (default "plain")

>    --insecure-sha1-intermediate  allows the use of insecure TLS protocols, such as SHA-1
>    --insecure-tls-version       allows to use TLS versions 1.0 and 1.1

Reviewer 2: nit:
```suggestion
   --insecure-sha1-intermediate  allows the use of insecure TLS protocols, such as SHA-1
   --insecure-tls-version        allows to use TLS versions 1.0 and 1.1
```

>    --insecure-tls-version         allows to use TLS versions 1.0 and 1.1

Reviewer: ```suggestion
   --insecure-tls-version       allows to use TLS versions 1.0 and 1.1
```
